PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -123,7 +192,7 @@ struct HfFragmentationFunctionTask {
         double z_parallel = (jetVector * d0Vector) / (jetVector * jetVector);
 
         // calculating angular distance in eta-phi plane
-        axisDistance = sqrt(pow(jet.eta() - d0Candidate.eta(), 2) + pow(jet.phi() - d0Candidate.phi(), 2));
+        axisDistance = RecoDecay::sqrtSumOfSquares(jet.eta() - d0Candidate.eta(), DeltaPhi(jet.phi(), d0Candidate.phi()));
```

Comment:
```suggestion
        axisDistance = RecoDecay::sqrtSumOfSquares(jet.eta() - d0Candidate.eta(), deltaPhi(jet.phi(), d0Candidate.phi()));
```
and everywhere else

Reply:
Don't know how I didn't notice that, thanks